static/app/types/workflowEngine/dataConditions.tsx

```diff
@@ -68,12 +52,19 @@ export enum DataConditionGroupLogicType {
   NONE = 'none',
 }
 
+export const enum DetectorPriorityLevel {
```

Comment:
FYI, this enum is unused so I’ve deleted it in:

- https://github.com/getsentry/sentry/pull/92010

Please don’t add unused code that is needed in the future because it is indistinguishable from dead code that can be deleted - try to add code at the time when it is used instead.